### DISCUSSION TITLE
Support (counter/icon) badges on entity icons and cards

### DISCUSSION THREAD

**Discussion by nagtegaal** [Clooos/Bubble-Card#622] [Feature requests]
This is a feature request to have the ability to support badges on the entity icons and cards...
Pretty straight forward question imo, perhaps hard to implement. But, would be a very nice addition

**Comment by Clooos** [comment]
Hard indeed, but you are not the first to ask for this feature so I keep this in my mind. For now you can add sub-buttons to display more informations.
